.github/workflows/tests.yml

```
@@ -58,7 +58,7 @@ jobs:
       - name: Install testing dependencies
         working-directory: services/http/api
         run: yarn
-      - name: Build test
+      - name: Building Tests
```

This should be kept consistent with the other names, so Capitalize the first letter only and keep the verb tense consistent.
```suggestion
      - name: Build tests
```

---

Understood!